python/tests/roundtriputils.py

```diff
@@ -0,0 +1,202 @@
+import io
+import pathlib
+import subprocess
+import types
+from typing import Callable, TypeVar, cast
+
+import test_model as tm
+from .factories import Format
+from .factories import get_reader_writer_types
+
+# pyright: basic
+
+
+_translator_path = (
+    pathlib.Path(__file__).parent / "../../cpp/build/translator"
```

Comment:
This path depends on the `_package` file location for cpp generation. So if `cpp` is renamed to `cplusplus` it fails. Not a big deal, but since that package file is yaml, we could read it from there.

Reply:
It also has something to do with the repo structure and where and what we call the cmake `build` directory and where the binary ultimately is. And the model directory is another variable if we use the _package.yml file